src/main/java/com/matt/forgehax/mods/ElytraFlight.java

```diff
@@ -27,53 +34,183 @@
           .description("Start flying when enabled")
           .defaultTo(false)
           .build();
-  
+
   public final Setting<Double> speed =
       getCommandStub()
           .builders()
           .<Double>newSettingBuilder()
           .name("speed")
-          .description("Movement speed")
+          .description("Base flight speed")
+          .defaultTo(0.0825D)
+          .build();
+
+  public final Setting<Float> boost =
+      getCommandStub()
+          .builders()
+          .<Float>newSettingBuilder()
+          .name("boost")
+          .description("Acceleration amount")
+          .defaultTo(0.05F)
+          .build();
+
+  public final Setting<Double> maxboost =
+      getCommandStub()
+          .builders()
+          .<Double>newSettingBuilder()
+          .name("maxboost")
+          .description("Max speed in control mode")
+          .defaultTo(1.60D)
+          .build();
+  
+  public final Setting<Double> up_speed =
+      getCommandStub()
+          .builders()
+          .<Double>newSettingBuilder()
+          .name("up_speed")
```

Comment:
Use `-`  over `_` in command names (it's okay to use it in the field name)

Reply:
Nice! Will change it also in other places